### DISCUSSION TITLE
Windows ARM64 Support

### DISCUSSION THREAD

**Discussion by slannon97** [BearWare/TeamTalk5#2446] [Ideas]
### Description

Now that Windows computers are being released with ARM processors, Team Talk should have a native ARM64 version. Currently, the application has to run with emulation, and it would be nice to have a native version for this version of Windows.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
I need someone to sponsor an ARM64 PC with Windows. I only have Intel hardware.
